### DISCUSSION TITLE
ScriptReference/AssetDatabase.LoadAssetAtPath

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#14] [Notes]
# ScriptReference/AssetDatabase.LoadAssetAtPath

https://docs.unity3d.com/ScriptReference/AssetDatabase.LoadAssetAtPath.html

**Comment by vertxxyz** [comment]
If you're trying to load an asset from a Package you can use `"Packages/com.example.example/..."`, regardless of the *real* location of the package files.

**Comment by vertxxyz** [comment]
Paths must be local to an Assets or Packages folder. You can get a local path to the Assets folder from a full path using:
```csharp
path = path[(Application.dataPath.Length - "Assets".Length)..];
```
(This code presumes the provided path will always be rooted to the correct drive)
